### DISCUSSION TITLE
Opening rendered qmd file in Presentation pane not in Viewer in RStudio

### DISCUSSION THREAD

**Discussion by Andrzej-Andrzej** [quarto-dev/quarto-cli#6022] [Q&A]
### Description

Hi,

When I render qmd file into presentation Quarto revealjs,  it opens in Viewer pane. What to do to have it in Presentation pane ? Somehow it worked a few weeks ago but I do not know why it is not working anymore.

I am on windows 10, quarto --version = 1.3.361, latest daily RStudio 2023.08.0 Build 53.

There is a disscussion about it here below but what is the final conclusion I do not know. 

[https://github.com/rstudio/rstudio/issues/11609](https://github.com/rstudio/rstudio/issues/11609)

**Comment by cderv** [comment]
I am on Windows too with 2023.06.0 Build 421 (last release) and it is working. 

I tested with  2023.08.0 build 60 and it is working too. 

Revealjs output shows in Presentation pane. 

If this does not work for you, I would suggest try isolate the behavior and open an issue in RStudio IDE. I don't think this is related to Quarto itself.

**Reply by Andrzej-Andrzej** [reply]
Thank you, how can I isolate this ?

**Reply by mcanouil** [reply]
You can try other examples, examples located in a different directory, double check your RStudio settings, _etc._

**Reply by Andrzej-Andrzej** [reply]
Thank you, I have checked and other examples are working properly This is strange why does one specific file is not working that way and opens in a viewer ?

**Reply by cderv** [reply]
> This is strange why does one specific file is not working that way and opens in a viewer ?

If you can share this specific file we can have a look. Otherwise, it will be hard to tell
